### PR TITLE
Remove RadioButtons React component from Storybook.

### PR DESCRIPTION
**Pull request by jamigibbs**
## Chromatic

https://1114-remove-radiobuttons-react--60f9b557105290003b387cd5.chromatic.com

## Description
As part of the deprecation work in [#1114 ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1114), this PR removes the RadioButtons React component from Storybook.

## Testing done
Storybook

## Screenshots
Screen Shot 2022-10-05 at 1 06 15 PM

## Acceptance criteria
- [ ] The RadioButtons React component is removed from Storybook.

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by k80bowman** [comment]
This looks great, just make sure you use the "major" label instead of "ignore-for-release" so we get this in the release notes.

**Comment by k80bowman** [comment]
I was just looking at this again and I wasn't thinking straight earlier. We can't delete the React component yet, there are a lot of applications still using it. We can remove the Storybook story file and the listing from the additional-docs file, but that's it. And in that case, the label should be ignore-for-release.

**Comment by jamigibbs** [comment]
@k80bowman Oh geez that's my bad. Even though I specifically wrote this was just for Storybook, I still removed it from the component library probably getting it mixed up with #1205.

File: packages/react-components/src/components/RadioButtons/RadioButtons.mdx [hunk 1]
```diff
@@ -92,4 +92,4 @@ export class RenderedComponent extends React.Component {
   }
 }
 
-<RenderedComponent />
+<RenderedComponent />
```
Comment (k80bowman) [inline packages/react-components/src/components/RadioButtons/RadioButtons.mdx:95]: I know this isn't actually changing anything, but is there any way to remove this change?

Comment (jamigibbs) [reply packages/react-components/src/components/RadioButtons/RadioButtons.mdx]: It seemed to want a line break at the end so this should be resolved now.

**Review by k80bowman — APPROVED** [review]

**Pull request merged**